### ISSUE TITLE
Files in .csproj with \ not found

### ISSUE THREAD

**Issue by Alisis33** [radimitrov/CSharpShellApp#205]
if files in .csproj have paths with \ instead of /, this files are not found.

(examples from xamarin-forms-samples must be manualy corrected. Like from other non cshell sources.)

**Comment by radimitrov** [comment]
Yes, problematic. Especially since all path related operations need a "/" under Android.
I will fix it. However at least for now any edits to the related file paths will change their "\" to "/".
